packages/mint-components/src/components/sqm-titled-section/readme.md

```diff
@@ -7,13 +7,29 @@
 
```

Comment:
Flagging that the title for this readme is wrong

Reply:
Changed to `# sqm-titled-section`